functions.php

```diff
@@ -550,6 +556,7 @@ function exhibit_builder_items_browse_sql($args)
     $db = get_db();
 
     $exhibit = isset($params['exhibit']) ? $params['exhibit'] : null;
+    $exhibit_page = isset($params['exhibit-page']) ? $params['exhibit-page'] : null;
```

Comment:
This (along with the conditional checking for it down on 591) doesn't look like it's used anywhere.